packages/fast-components/src/switch/switch.ts

```diff
@@ -0,0 +1,104 @@
+import { attr, observable } from "@microsoft/fast-element";
+import { FormAssociated } from "../form-associated";
+import { keyCodeSpace } from "@microsoft/fast-web-utilities";
+import { bool } from "../utilities";
+import { type } from "os";
```

Comment:
where is this coming from?

Reply:
Yeah that was one of those VS Code auto import situations... Thanks!